include/onnx-mlir/Runtime/OMTensor.h

```diff
@@ -167,6 +167,18 @@ OM_EXTERNAL_VISIBILITY void omTensorDestroy(OMTensor *tensor);
  */
 OM_EXTERNAL_VISIBILITY void *omTensorGetDataPtr(const OMTensor *tensor);
 
+/**
+ * \brief OMTensor allocated data pointer getter.
+ *
+ * @param tensor pointer to the OMTensor
+ * @return pointer to the allocated memory buffer of the OMTensor,
+ *         This should only be used when needing needing to create
```

Comment:
Two `needing`.

Reply:
tx, added some additional comments on the difference between data and allocated pointers.